gateway-service/src/main/resources/application.yml

```diff
@@ -113,13 +114,6 @@ server:
         trustStore: keystore/localhost/localhost.truststore.p12
         trustStorePassword: password
         trustStoreType: PKCS12
-
-    cloud:
-        gateway:
-            discovery:
-                locator:
-                    enabled: false
-                    lowerCaseServiceId: true
```

Comment:
just out of curiosity, this wasn't doing anything, or is it related to this feature?

Reply:
It wasn't doing anything, so, it's useless. I found it, when I was searching for the parameter.